package.json

```diff
@@ -1,19 +1,24 @@
 {
-  "name": "uni-registrar-driver-did-example",
+  "name": "uni-registrar-driver-did-factom",
```

Comment:
See remark about official driver. Also applies to description

Reply:
Updated.